Makefile

```diff
@@ -260,11 +260,15 @@ security-test:
 	go test ${T} ./security/pkg/...
 	go test ${T} ./security/cmd/...
 
+.PHONY: mixer-client-test
+mixer-client-test:
+	mixer/test/client/run_test.sh
```

Comment:
Offhand it seems like the 1-line functionality of run_test.sh should be a good candidate to instead implement as a rule within make.

Reply:
try to make "go test ./mixer/.." works for the new tests: ( need to support running the tests in parallel).  Once it works, this rule will be removed